scripts/linux/buildBranch.sh

```diff
@@ -25,7 +25,7 @@ SRC_E2E_TEST_FILES_DIR=$ROOT_FOLDER/e2e_test_files
 SRC_CERT_TOOLS_DIR=$ROOT_FOLDER/tools/CACertificates
 FUNCTIONS_SAMPLE_DIR=$ROOT_FOLDER/edge-modules/functions/samples
 VERSIONINFO_FILE_PATH=$BUILD_REPOSITORY_LOCALPATH/versionInfo.json
-
+OS=
```

Comment:
I don't see this being used below. Is it supposed to be:

```suggestion
DOTNETBUILD_OS=
```